### PR TITLE
Update data source for `Projects built on Ethereum` in `what-is-ethereum.tsx`

### PR DESCRIPTION
**Pull request by MahendraBishnoi29**
## Description
remove broken link 

## Related Issue
#9411

### CODE REVIEW

File: src/pages-conditional/what-is-ethereum.tsx [hunk 1]
```diff
@@ -524,7 +524,7 @@ const WhatIsEthereumPage = ({
                       <Tooltip
                         content={tooltipContent({
                           apiUrl:
-                            "https://www.stateofthedapps.com/stats/platform/ethereum#new",
+                            "#",
```
Comment (pettinarip) [inline src/pages-conditional/what-is-ethereum.tsx:527]: A link for this looks kind of odd. Could we modify the `tooltipContent` function to only render a Link if an `apiUrl` is provided?

**Review by pettinarip — COMMENTED** [review]
Cool, thanks for the PR @MahendraBishnoi29

File: src/pages-conditional/what-is-ethereum.tsx (hunk 1 above)
Comment (pettinarip) [reply src/pages-conditional/what-is-ethereum.tsx:527]: Ignore this @MahendraBishnoi29 pls 🙏🏼 

As you suggested, lets
- update the resource here to be https://dappradar.com/rankings/protocol/ethereum
- update the metric to be `3882`

Thanks!

Comment (MahendraBishnoi29) [reply src/pages-conditional/what-is-ethereum.tsx:527]: metric 🤔 @pettinarip

**Comment by konopkja** [comment]
What is the status of this PR? would like to change the number and update to the latest: 3986
ping @corwintines

**Comment by MahendraBishnoi29** [comment]
> 

I was waiting for someone's review @konopkja  🤔. I made the changes according to @pettinarip

**Comment by konopkja** [comment]
> > 
> 
> I was waiting for someone's review @konopkja 🤔. I made the changes according to @pettinarip

okay, sorry for the delay - the team has been notified and should look at this soon, thank you for your patience :)

**Review by corwintines — APPROVED** [review]

**Review by corwintines — APPROVED** [review]

**Pull request merged**